### PR TITLE
:memo: [ci skip] add notes on 'apm' path in Windows

### PR DESCRIPTION
**Pull request by BrainCrumbz**
Comments came out while discussing issue #11137

### CODE REVIEW

**Comment by damieng** [comment]
Given tests on Windows are still broken right now (it's my current task fixing them up) I'm not sure we want to encourage running them just yet.  Recommending we hold off on this for now.

**Comment by BrainCrumbz** [comment]
Fair enough. We just should remember to track this together with potential further Windows issues that could arise in #11543.

Side note: out of curiosity, how come a documentation issue is labeled as *api* ?

Cheers

File: docs/build-instructions/windows.md [hunk 1]
```diff
@@ -48,6 +48,17 @@ This will create the Atom application in the `out\Atom` folder as well as copy i
   * `--no-install` - Skips the installation task after building.
   * `--verbose` - Verbose mode. A lot more information output.
 
+### Running tests
+
+In order to run tests from command line you need `apm`, available after you install Atom or after you build that from scratch. If you installed it, run the following commands (assuming `C:\atom` is the root path of your Atom sources):
```
Comment (lee-dohm) [inline docs/build-instructions/windows.md]: * build that from scratch &rarr; build from source
* root path of your Atom sources &rarr; root of your Atom repository

File: docs/build-instructions/windows.md [hunk 2]
```diff
@@ -48,6 +48,17 @@ This will create the Atom application in the `out\Atom` folder as well as copy i
...
+
+```bash
+cd C:\atom
+apm test
+```
+
+When building Atom from scratch instead, the `apm` command is not added to the system path by default. In this case, you can either add it yourselves or you can explicitly list the complete path in previous commands. Anyway, its default location on a Win 64-bit installation is `"C:\Program Files (x86)\Atom\resources\cli\"`.
```
Comment (lee-dohm) [inline docs/build-instructions/windows.md]: * building Atom from scratch instead &rarr; building Atom from source
* add it yourselves &rarr; add it yourself
* or you can explicitly &rarr; or explicitly

**Comment by damieng** [comment]
I've tackled a bunch so far - paths that need normalizing, some font sizings and bufferd-process. Still a bunch to deal.

**Comment by BrainCrumbz** [comment]
Hi.
Does it makes sense to solve conflicts on this PR?

**Comment by damieng** [comment]
Go ahead and solve the conflicts. Can you add a note that there are still some Windows tests that don't pass as well?  I'll get it merged in. Thanks!

**Comment by BrainCrumbz** [comment]
You're welcome!  
There you go: please check if that looks fine.

File: docs/build-instructions/windows.md [hunk 3]
```diff
@@ -36,6 +36,19 @@ To also install the newly built application, use `script\build --create-windows-
...
+### Running tests
+
+In order to run tests from command line you need `apm`, available after you install Atom or after you build from source. If you installed it, run the following commands (assuming `C:\atom` is the root of your Atom repository):
+
+```bash
+cd C:\atom
+apm test
+```
+
+When building Atom from source, the `apm` command is not added to the system path by default. In this case, you can either add it yourself or explicitly list the complete path in previous commands. Anyway, its default location on a Win 64-bit installation is `"C:\Program Files (x86)\Atom\resources\cli\"`.
```
Comment (50Wliu) [inline docs/build-instructions/windows.md]: This is actually incorrect.  The new path is `%localappdata%\Atom\app-dev\resources\cli\`.

Comment (damieng) [inline docs/build-instructions/windows.md]: Just checked in a fix for that.

**Pull request merged**